src/libs/ReportUtils.js

```diff
@@ -2313,8 +2330,9 @@ export {
     isMoneyRequestReport,
     chatIncludesChronos,
     getAvatar,
+    getAvatarUrl,
     isDefaultAvatar,
-    getOldDotDefaultAvatar,
+    getDefaultAvatarURL,
```

Comment:
IMO these methods are in the wrong file. Can you move all of the avatar-related functions (that have nothing to do with reports) to `UserUtils.js` please?